modules/generators/content/password_file/secgen_local/local.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/ruby
+require_relative '../../../../../lib/objects/local_string_generator.rb'
+require 'erb'
+require 'fileutils'
+class PasswordFileGenerator < StringGenerator
+  attr_accessor :website_sample
+  attr_accessor :username_sample
+  attr_accessor :pass_list
+  LOCAL_DIR = File.expand_path('../../',__FILE__)
+  TEMPLATE_PATH = "#{LOCAL_DIR}/templates/password_file.md.erb"
+
+  def initialize
+    super
+    self.pass_list = Array.new
+  end
+  
+  def get_options_array
+    super + [['--password1', GetoptLong::OPTIONAL_ARGUMENT]]
+  end
+
+  def process_options(opt, arg)
```

Comment:
You _could_ use a "interests" and/or "malicious_interests" value to theme it:

Take a look in here for some examples: https://github.com/cliffe/SecGen/blob/master/modules/generators/sqlite/firefox_places_history_bookmarks/secgen_local/local.rb